agent-codeless/build.gradle

```diff
@@ -30,6 +30,23 @@ def instrumentationVersion = '0.14.9'
 
 def shadowPrefix = 'com.microsoft.applicationinsights.agentc.shadow'
 
+sourceSets {
+    devtest {
+        java {
+            srcDir file('src/devtest/java')
```

Comment:
I don't see this directory?

Reply:
hmm, i'm not sure why these aren't showing up for you. `agent-codeless/src/devtest/java`

Reply:
Same, github UI failing me (and I'm failing it).